### ISSUE TITLE
f.c6_3_41.FWscHIST.ne30_L58_exner.002

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#98]
**Description:**
Same as https://github.com/NCAR/amwg_dev/issues/53 + second set of modifications to SE to alleviate numerical noise near topography

+ [Map PHIS from GLL grid to physics grid](https://github.com/ESCOMP/CAM/issues/552) to alleviate noise in derived fields such as PSL and eddy Z500)

+ Exner function formulation of pressure gradient force (PGF) + subtract reference profiles from T and dp before applying hyperviscosity  (see [complete description](https://github.com/NCAR/amwg_dev/issues/96))

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/f.c6_3_41.FWscHIST.ne30_L58_exner.002

On github:
https://github.com/NCAR/amwg_dev/tree/f.c6_3_41.FWscHIST.ne30_L58_exner.002

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cam6_3_041

On github:
https://github.com/ESCOMP/CAM/tree/cam6_3_041
hash: 963ebdc

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/FWscHIST/f.c6_3_41.FWscHIST.ne30_L58_exner.002/atm/](https://webext.cgd.ucar.edu/FWscHIST/f.c6_3_41.FWscHIST.ne30_L58_exner.002/atm/)

**Contacts:**
@PeterHjortLauritzen, @adamrher, @JulioTBacmeister, @cecilehannay

**Comment by cecilehannay** [comment]
@PeterHjortLauritzen and @adam: this run completed 10 years. I added the diags: https://webext.cgd.ucar.edu/FWscHIST/f.c6_3_41.FWscHIST.ne30_L58_exner.002/atm/

**Comment by adamrher** [comment]
See here 10 year means and variance of FT (lev=50) and OMEGA500:

![temp_paper FT refpsens](https://user-images.githubusercontent.com/29961476/162580231-f9611f79-4593-4712-93c1-6fe397db0c53.png)

![temp_paper OMEGA500 refpsens](https://user-images.githubusercontent.com/29961476/162580234-a43a46b4-ba41-445c-937e-5436c3c4d064.png)

A couple observations
- There's a back-buldge feature (geology term) on the back side of the Andes in the exner only run. It can be seen in means/variance of FT and the mean of OMEGA500. Not sure if this is a gibbs ringing type phenomena, but it is eliminated in exner+refp simulation, for the most part.
- There's a similar pos-neg-pos wave over the Himalayas in the exner only run, as seen in the mean OMEGA500 plot. The exner+refp eliminates this and creates a single local maxima aligned with the front range. This seems like a more physical solution to me.

I see that we PRECC/PRECL on the se-grid as well. I will add those plots to this thread later.

**Comment by adamrher** [comment]
10 year mean/variance of PRECT:

![temp_paper PRECT refpsens](https://user-images.githubusercontent.com/29961476/162581961-b8f5b11d-e7d1-405b-a2a5-8fdf67b287ac.png)

Note the reduction in spurious precip around the Peruvian bend in both the means and variance. I think it's clear the exner+refp is the better model. As peter says, refp is more effective at mitigating the spurious upslope transport that is likely related to the Peruvian precip problem.

**Comment by PeterHjortLauritzen** [comment]
I agree with Adam! PR is being prepared with the modifications for "exner+refp". Please use these source mods in any future simulations until the PR has been approved and merged with cam_development. Thank you!

**Comment by justin-richling** [comment]
@PeterHjortLauritzen, @adamrher, @JulioTBacmeister, @cecilehannay

ADF diags: [https://webext.cgd.ucar.edu/FWscHIST/f.c6_3_41.FWscHIST.ne30_L58_exner.002/atm/ADF/](https://webext.cgd.ucar.edu/FWscHIST/f.c6_3_41.FWscHIST.ne30_L58_exner.002/atm/ADF/)
